ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -558,6 +566,9 @@ open class DrawerController: UIViewController {
     }
 
     open override func accessibilityPerformEscape() -> Bool {
+        if delegate?.shouldDismissDrawer?(self) == false {
```

Comment:
this seems bit weird to me. for most of the clients, this optional protocol is not implemented. So you are checking if nil is equal to false. guard let foo = protocol else { return } might be cleaner

Reply:
I am seeing that `nil == false` is returning `false`, hence this is working fine.
![Screenshot 2020-09-23 at 8 48 58 AM](https://user-images.githubusercontent.com/25812641/93964633-c1a86780-fd7d-11ea-9029-1d606baea725.png)

The challenge with using `guard` here is that then I'll need to write dismiss drawer statement twice like below:
![Screenshot 2020-09-23 at 8 52 41 AM](https://user-images.githubusercontent.com/25812641/93964678-e4d31700-fd7d-11ea-9043-d8ed272413b6.png)

I don't have a strong opinion here whether to keep if else or use guard here. I feel introducing guard is brining unnecessary complication here. Let me know what do you think?

Reply:
created `dismissPresentingViewController` to keep code cleaner.